src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -0,0 +1,151 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.arguments import CLIArgumentType
+
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azure.cli.core.commands.parameters import (get_location_type, get_enum_type, tags_type)
+from azure.cli.command_modules.ams._completers import get_role_definition_name_completion_list
+
+from azure.mediav3.models import (EncoderNamedPreset, Priority, AssetContainerPermission)
+
+from ._validators import storage_account_id
+
+# pylint: disable=line-too-long
+
+
+def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statements
+    name_arg_type = CLIArgumentType(options_list=('--name', '-n'), metavar='NAME')
+    account_name_arg_type = CLIArgumentType(options_list=('--account-name', '-a'), metavar='ACCOUNT_NAME')
+    storage_account_arg_type = CLIArgumentType(options_list=('--storage-account'), metavar='STORAGE_NAME')
+    password_arg_type = CLIArgumentType(options_list=('--password', '-p'), metavar='PASSWORD_NAME')
+    transform_name_arg_type = CLIArgumentType(options_list=('--transform-name', '-t'), metavar='TRANSFORM_NAME')
+    expiry_arg_type = CLIArgumentType(options_list=('--expiry'), metavar='EXPIRY_TIME')
```

Comment:
Don't use the tuple syntax.  Instead use list syntax. 
```Python
options_list=('--foo', '-f)  # don't do this
options_list=['--foo', '-f'] # do this
```

Where did you see the use of tuple syntax?  In help docs or another module?

Reply:
Ok, we will apply this improve. 

We followed the pattern from role module: https://github.com/Azure/azure-cli/blob/42b03e9c3c7114c0768d8ef8212023f480ad5c85/src/command_modules/azure-cli-role/azure/cli/command_modules/role/_params.py#L16.

Reply:
Done